hatice-ecevit-2016400138/assignment1/Anagram.java

```diff
@@ -0,0 +1,69 @@
+ import java.util.Arrays;
+ import java.util.HashMap;
+public class Anagram {
+
+    public static void main(String[] args) {
+        System.out.println(areAnagrams("apple","appll",true));
+        System.out.println(areAnagrams("listen","Silent",false));
+        System.out.println(areAnagramSentences("hi there hello","iH tHree lloeh",false));
+    }
+    public static boolean areAnagrams(String s1, String s2, boolean caseSensitive){
+        if(!caseSensitive){
+            s1=s1.toLowerCase();
+            s2=s2.toLowerCase();
+        }
+        HashMap<Character,Integer> letterMap=new HashMap<Character, Integer>();
+        if (s1.length()!=s2.length())
+                return false;
+        else{
+            for(int i=0;i<s1.length();i++){
+                char currentChar=s1.charAt(i);
+                if(letterMap.containsKey(s1.charAt(i))){
```

Comment:
Optional:
Currently, you call `put()` on both the `then`- and `else`-branch.
You could simplify this slightly by having `value` default `0` if not present in the map already.